src/app/util/af-event.cpp

```diff
@@ -91,6 +120,23 @@ EmberEventData emAfEvents[] = {
     { NULL, NULL }
 };
 
+void EventControlHandler(chip::System::Layer * systemLayer, void * appState, chip::System::Error error)
+{
+    EmberEventControl * control = reinterpret_cast<EmberEventControl *>(appState);
+    if (control->status != EMBER_EVENT_INACTIVE)
+    {
+        for (size_t i = 0; i < sizeof(emAfEvents); i++)
```

Comment:
This `sizeof()` makes no sense to me, unless each entry in `emAfEvents` is one-byte, which it's not.

How about doing it like so instead:
```
for (auto& event : emAfEvents) 
{
  if (event.control == control)
  {
    control->status = EMBER_EVENT_INACTIVE;
    event.handler();
    return;
  }
}
```
or so?

Reply:
Ah right. I always forgot C++11. Will do.

Reply:
Done.